packages/models/src/bitcoin-unit.model.ts

```diff
@@ -0,0 +1,46 @@
+import BigNumber from 'bignumber.js';
+
+export enum WalletBitcoinUnit {
+  bitcoin = 'bitcoin',
+  millibit = 'millibit',
+  bit = 'bit',
+  satoshi = 'satoshi',
+  millisatoshi = 'millisatoshi',
+}
+export type BitcoinUnit = keyof typeof WalletBitcoinUnit;
+
+export type BitcoinUnitSymbol = 'BTC' | 'mBTC' | 'μBTC' | 'sat' | 'msat';
+
+export interface BitcoinUnitInfo {
+  name: WalletBitcoinUnit;
+  symbol: BitcoinUnitSymbol;
+  decimal: BigNumber;
+}
+
+export const bitcoinUnitsKeyedByName: Record<WalletBitcoinUnit, BitcoinUnitInfo> = {
+  [WalletBitcoinUnit.bitcoin]: {
+    name: WalletBitcoinUnit.bitcoin,
+    symbol: 'BTC',
+    decimal: BigNumber(1),
+  },
+  [WalletBitcoinUnit.millibit]: {
+    name: WalletBitcoinUnit.millibit,
+    symbol: 'mBTC',
+    decimal: BigNumber(0.001),
+  },
+  [WalletBitcoinUnit.bit]: {
+    name: WalletBitcoinUnit.bit,
+    symbol: 'μBTC',
+    decimal: BigNumber(0.000001),
+  },
+  [WalletBitcoinUnit.satoshi]: {
+    name: WalletBitcoinUnit.satoshi,
+    symbol: 'sat',
+    decimal: BigNumber(0.000001),
+  },
```

Comment:
satoshi is the smallest unit, so afaik this is wrong, and there is no millisatoshi?

Reply:
Meant to put this here: https://en.bitcoin.it/wiki/Units

Reply:
![Screenshot 2024-09-13 at 10 24 04 AM](https://github.com/user-attachments/assets/86665df3-61d3-4952-92c3-ca338d08bc27)